### PR TITLE
[receiver/filelog] Add note on tracking symlinked files for `poll_interval` setting.

### PR DESCRIPTION
**Pull request by ycombinator**
**Description:**

This PR updates documentation for the `poll_interval` setting to mention how it should be set when the `filelog` receiver is tracking symlinked files.

**Related:**  https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32292

### CODE REVIEW

File: receiver/filelogreceiver/README.md [hunk 1]
```diff
@@ -16,45 +16,45 @@ Tails and parses logs from files.
...
+| `include_file_owner_name`           | `false`                              | Whether to add the file owner name as the attribute `log.file.owner.name`. Not supported for windows.                                                                                                                                                                        |
+| `include_file_owner_group_name`           | `false`                              | Whether to add the file group name as the attribute `log.file.owner.group.name`. Not supported for windows.                                                                                                                                                                  |
+| `poll_interval`                     | 200ms                                | The [duration](#time-parameters) between filesystem polls. If the receiver is being used to track a symlinked file and the symlink target is expected to change frequently, make sure to set the value of this setting to something lower than the symlink update frequency. |
```
Comment (tiffany76) [inline receiver/filelogreceiver/README.md:35]:
```suggestion
| `poll_interval`                     | 200ms                                | The [duration](#time-parameters) between filesystem polls. If the receiver is being used to track a symlinked file and the symlink target is expected to change frequently, make sure to set the value of this setting lower than the symlink update frequency. |
```

**Review by tiffany76 — COMMENTED** [review]
Just a small suggestion from a docs perspective.

**Review by tiffany76 — APPROVED** [review]
LGTM.

**Comment by ycombinator** [comment]
>  This seems more like a troubleshooting tip. Maybe we should start a troubleshooting section lower in the doc?

Done in 6fac1a46cdeb766a2c31b3fc306cbfc5107eca99.

**Review by djaglowski — APPROVED** [review]

**Review by tiffany76 — APPROVED** [review]
LGTM.

**Pull request merged**